github_pipeline_observability/change.finished.md

```diff
@@ -0,0 +1,35 @@
+### Change event for Semantic Dictionary
+
+`task.change.[open/merged/closed]`
+
+| Property | Value | Description | 
+|---|---|---|
+| event.id  |   |
+| event.kind  | `SDLC_EVENT`  | x 
+| event.version | `0.1.0` |  x
+| event.category | `task` |  x
+| event.type | `change` |  x
+| event.provider | `github.com` |  x
+| event.status |  [`merged`, `closed`, `open`] |  x
+| duration | *merged_at - created_at* |  x
+| start_time | *created_at* |  x
+| end_time | *merged_at or closed_at* |  x
+| timestamp | |  x
+| | | |
+| task.id | `871023010` | The unique task identifier.
+| task.name | `docs: Update of Readme.md` | A human-readable name for a task.
+| task.outcome | `success` | Outcome of a task run.
+| | | |
+| vcs.change.id  | `4711` | The identifier of the change, for example, pull request ID or merge request ID. It is typically unique per repository and generated by the version control system itself.
+| vcs.change.title  | `docs: Update of Readme.md`  | The human-readable title of the change, for example, pull request title or merge request title.  **Attention:** Ident to: "task.name"
```

Comment:
@stefaneberl The OTel and our SemDic spec define the property `vcs.change.title`. This would be a duplicate of `task.name` (see value). Should we duplicate the information?

Reply:
We accept a duplication